documentation/sphinx/source/administration.rst

```diff
@@ -488,6 +488,19 @@ If a process has had more than 10 TCP segments retransmitted in the last 5 secon
 
       10.0.4.1:4500       ( 3% cpu;  2% machine; 0.004 Gbps;  0% disk; REXMIT! 2.5 GB / 4.1 GB RAM  )
 
+Machine-readable status
+--------------------------------
+
+The status command can provide a complete summary of statistics about the cluster and the database with the ``json`` argument. Full documentation for ``status json`` output can be found :doc:`here <mr-status>`.
+From the output of ``status json``, operators can find useful health metrics to determine whether or not their cluster is hitting performance limits.
+
+====================== =========================================================== ===================================================
+Ratekeeper limit        ``cluster.qos.transactions_per_second_limit``               The number of read versions per second that the cluster can give out. A low ratekeeper limit indicates that the cluster performance is limited in some way. The reason for a low ratekeeper limit can be found at ``cluster.qos.performance_limited_by``. ``cluster.qos.released_transactions_per_second`` describes the number of read versions given out per second, and can be used to tell how close the ratekeeper is to throttling.
+Storage queue size      ``cluster.qos.worst_queue_bytes_storage_server``            The maximum size in bytes of a storage queue. Each storage server has mutations that have not yet been made durable, stored in the storage queue. If this queue gets too large for any storage server, the ratekeeper will increase throttling. Storage servers with large storage queues have fallen behind and will likely struggle to serve read requests.
```

Comment:
Ratekeeper uses the `limiting_queue_bytes_storage_server` to throttle, which depending on your configuration may be different than the worst bytes. The main things to watch out for with large worst bytes are that it signals you may have a problem somewhere and the note about falling behind. I think this is a distinction that may cause confusion if it isn't understood by someone monitoring them.

The main idea behind it is that ratekeeper allows storage servers from one fault domain to fall behind in some configurations. This avoid scenarios where a single storage server failure drags down the cluster. This metric exists because it's useful to know what's happening on the worst process, while the limiting metric is useful if you want to understand the impacts on ratekeeper.